### PR TITLE
Changed MSTeams Sync to MS Teams Connector

### PR DESCRIPTION
**Pull request by cwarnermm**

### CODE REVIEW

File: webapp/src/components/appManifestSetting.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ export default class MSTeamsAppManifestSetting extends React.PureComponent<Props
                     {'Clicking the Download button below will generate an application manifest that will embed this instance of Mattermost. '}
                 </p>
                 <p>
-                    {'Mattermost embedded in MS Teams can be used together with MSTeams Sync, or independently.'}
+                    {'Mattermost embedded in MS Teams can be used together with MS Teams Connect, or independently.'}
```
Comment (cwarnermm) [inline webapp/src/components/appManifestSetting.tsx:24]:
```suggestion
                    {'Mattermost embedded in MS Teams can be used together with MS Teams Connector, or independently.'}
```

**Comment by johndavidlugtu** [comment]
@mkdbns can you add a dev reviewer from your team. Thanks!

File: docs/developer_docs.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ This plugin supports the following features:
 
 1. Go to the [releases page of this GitHub repository](github.com/mattermost/mattermost-plugin-msteams-sync/releases) and download the latest release for your Mattermost server.
 2. Upload this file on the Mattermost **System Console > Plugins > Management** page to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
-3. Enable the plugin from **System Console > Plugins > MSTeams Sync**.
+3. Enable the plugin from **System Console > Plugins > MS Teams Connector**.
```
[the remark this answers is not in the document]
Comment (manojmalik20) [reply docs/developer_docs.md:56]: The text is also present in the `azure_setup.md` file. Please search `teams sync` in the whole repo without case sensitivity to find all the occurences.

[the remark this answers is not in the document]
Comment (cwarnermm) [reply docs/developer_docs.md:56]: Thanks, @manojmalik20! I believe I've caught the remaining instances in my latest commit.

File: docs/developer_docs.md
[the remark this answers is not in the document]
Comment (cwarnermm) [reply docs/developer_docs.md:1]: Yes. I've updated that one too. The only reference I didn't update was a code-level comment.

**Review by manojmalik20 — APPROVED** [review]

**Comment by lieut-data** [comment]
@cwarnermm, we're still going to execute on this, but proposing we start from a fresh pull request. Tracking in https://mattermost.atlassian.net/browse/MM-55478.

**Pull request closed without merging**